src/os_tester/stages.py

```diff
@@ -1,29 +1,47 @@
 import sys
+
 from os import path
 from typing import Any, Dict, List
-
 import yaml  # type: ignore
 
-
-class stage:
+class subpath:
```

Comment:
```suggestion
class subPath:
```

Please use lowerCamelCase for class names.